qcodes/instrument_drivers/ZI/ZIHDAWG8.py

```diff
@@ -63,11 +63,15 @@ def __init__(self, name: str, device_id: str, **kwargs) -> None:
         self.warnings_as_errors: List[str] = []
         self._compiler_sleep_time = 0.01
 
-    def snapshot_base(self, update: bool = False,
-                      params_to_skip_update: Sequence[str] = None) -> Dict:
-        """ Override the base method to ensure all values are up-to-date"""
-        return super(ZIHDAWG8, self).snapshot_base(update=True,
-                                                   params_to_skip_update=None)
+    def snapshot_base(self, update: bool = True,
+                      params_to_skip_update: Sequence[str] = 'features_code') -> Dict:
```

Comment:
As this is a sequence of strings it should be `('features_code',)`

Reply:
That's right